Formula/open-zwave.rb

```diff
@@ -1,6 +1,6 @@
 class OpenZwave < Formula
   desc "Library for selected Z-Wave PC controllers"
-  homepage "http://www.openzwave.com"
```

Comment:
this is still valid

Reply:
True, thank you. This is one of those that fail the checker and apparently I missed this one on the manual double-check.